src/components/chat/chat-thread.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import classNames from "classnames";
+import { UserModelType } from "../../models/stores/user";
+import { WithId } from "../../hooks/firestore-hooks";
+import { CommentDocument } from "../../lib/firestore-schema";
+import { CommentCard } from "./comment-card";
+import { getToolContentInfoById } from "../../models/tools/tool-content-info";
+import UserIcon from "../../assets/icons/clue-dashboard/teacher-student.svg";
+import {ChatCommentThread} from "./chat-comment-thread";
+
+import "./chat-thread.scss";
+import { ToolIconComponent } from "./tool-icon-component";
+
+interface IProps {
+  user?: UserModelType;
+  activeNavTab?: string;
+  chatThreads?: ChatCommentThread[];
+  onPostComment?: (comment: string) => void;
+  onDeleteComment?: (commentId: string, commentContent: string) => void;
+  focusDocument?: string;
+  focusTileId?: string;
+}
+
+
+export const ChatThread: React.FC<IProps> = ({ activeNavTab, user, chatThreads, 
+  onPostComment, onDeleteComment,
+  focusDocument, focusTileId }) => {
+  // make focusId null if undefined so it can be compared with tileId below.
+  const focusId = focusTileId === undefined ? null : focusTileId;
+  const focusedItemHasNoComments = !chatThreads?.find(item => (item.tileId === focusId));
+  
+  return (
+    <div className="chat-list" data-testid="chat-list">
+         {chatThreads?.map((commentThread: ChatCommentThread, idx:number) => {
+            const title = commentThread.title ? commentThread.title : '';
+            const shouldShowUserIcon = 
+                commentThread.comments.some((comment: WithId<CommentDocument>) => user?.id === comment.uid);
+            const numComments = commentThread.comments.length; 
+            const shouldBeFocused = commentThread.tileId === focusId;
+            const Icon = commentThread.tileType && getToolContentInfoById(commentThread.tileType)?.Icon;
+            const key= commentThread.tileId ? commentThread.tileId : "document";
```

Comment:
So a document can have only one thread associated with it, right?

Reply:
That is my understanding, yes!

Reply:
Put another way, any document (curriculum or user), can have one thread associated with the document as a whole and a thread for any tile in the document.